bin/dry-run.rb

```diff
@@ -295,8 +295,16 @@
     Dependabot::Experiments.register(:enable_beta_ecosystems, true)
   end
 
-  opts.on("--cooldown", "Cooldown configuration as a JSON object") do |value|
-    $options[:cooldown] = JSON.parse(value)
+  opts.on("--cooldown JSON", "Cooldown configuration as a JSON object") do |value|
+    cooldown_options = JSON.parse(value)
```

Comment:
this seems like something @kbukum1 ran into a few weeks ago; are there other places where we need to do this conversion?

Also, can we add tests?

Reply:
Thanks for the comment @abdulapopoola 

I'll raise a separate PR introducing tests for `dry-run.rb`.

Reply:
Here is the aforementioned [PR](https://github.com/dependabot/dependabot-core/pull/11981).